requirements/default.txt

```diff
@@ -4,5 +4,5 @@ matplotlib>=2.0.0,!=3.0.0
 networkx>=2.0
 pillow>=4.3.0,!=7.1.0,!=7.1.1
 imageio>=2.3.0
-tifffile>=2019.7.26
+tifffile>=2020.5.30
```

Comment:
Q: should we leave the original dependency but add `tifffile>=2020.5.30` to the `docs` requirements?

Reply:
sounds good to me

Reply:
Ok, done.